ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -3185,7 +3185,7 @@ public void loadTable(Path loadPath, String tableName, LoadFileType loadFileType
     //column stats will be inaccurate
     if (resetStatistics) {
       LOG.debug("Clearing table statistics for " + tbl.getDbName() + "." + tbl.getTableName());
-      StatsSetupConst.clearColumnStatsState(tbl.getParameters());
+      StatsSetupConst.setBasicStatsState(tbl.getParameters(),StatsSetupConst.FALSE);
```

Comment:
Even during loadPartition, we need to reset stats in table properties as well which is missing now.